packages/ebs-helper/src/classes/HelixExtensionSecretList.ts

```diff
@@ -24,12 +24,15 @@ export class HelixExtensionSecretList extends DataObject<HelixExtensionSecretLis
 			.map(secret => secret.content);
 	}
 
-	verifyJwt(token: string): unknown {
+	async verifyJwt(token: string): Promise<JWTPayload> {
```

Comment:
I wasn't sure if you wanted this to remain as unknown, but `jose` provides a payload type. We can change it back if needed.
```suggestion
	async verifyJwt(token: string): Promise<unknown> {
```

Reply:
I looked at the type and it seems fine because pretty much everything is optional.